civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -6394,3 +6401,107 @@ function civihr_employee_portal_node_export_import_alter(&$nodes, $format, $save
     WebformTransferService::preImport($node);
   }
 }
+
+/**
+ * Creates the default reports configuratio in the database
+ */
+function _civihr_employee_portal_create_default_reports_configuration() {
+  $configurations = [
+    [
+      'report_name' => 'undefined',
+      'label' => 'FTE by Department',
+      'json_config' => 'null'
+    ],
+    [
+      'report_name' => 'people',
+      'label' => 'Gender by Level',
+      'json_config' => '{"menuLimit":"200","cols":["Role level type"],"rows":["Employee gender"],"exclusions":{"Role location":["Clapham"]},"inclusions":{"Role location":["Hammersmith","Islington","Stratford","Victoria"]},"unusedAttrsVertical":"false","autoSortUnusedAttrs":"false","rendererName":"Bar Chart","aggregatorName":"Count","inclusionsInfo":{"Role location":["Hammersmith","Islington","Stratford","Victoria"]}}'
+    ],
+    [
+      'report_name' => 'people',
+      'label' => 'Gender by Department',
+      'json_config' => '{"menuLimit":"200","cols":["Role department"],"rows":["Employee gender"],"exclusions":{"Role location":["Clapham"]},"inclusions":{"Role location":["Hammersmith","Islington","Stratford","Victoria"]},"unusedAttrsVertical":"false","autoSortUnusedAttrs":"false","rendererName":"Bar Chart","aggregatorName":"Count","inclusionsInfo":{"Role location":["Hammersmith","Islington","Stratford","Victoria"]}}'
+    ],
+    [
+      'report_name' => 'people',
+      'label' => 'Gender by Location',
+      'json_config' => '{"menuLimit":"200","cols":["Employee gender"],"rows":["Role location"],"unusedAttrsVertical":"false","autoSortUnusedAttrs":"false","rendererName":"Table","aggregatorName":"Count"}'
+    ],
+    [
+      'report_name' => 'undefined',
```

Comment:
Shouldn't this belong to the people report?
Does it have something to do with not having a json_config yet?

Reply:
This shouldn't have been included